core/src/main/scala/com/criteo/cuttle/Executor.scala

```diff
@@ -378,6 +377,32 @@ class Executor[S <: Scheduling] private[cuttle] (
                                          limit: Int): Seq[ExecutionLog] =
     queries.getExecutionLog(queryContexts, jobs, sort, asc, offset, limit).transact(xa).unsafePerformIO
 
+  private[cuttle] def getStats(jobs: Set[String]): Json = {
+    val (running, waiting) = runningExecutionsSizes(jobs)
```

Comment:
I just wanted to point out that provided figures can be inconsistent because stm accesses are non enclosed in a single 
transaction. Not use this is critical though